src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -39,46 +41,50 @@ public function __construct(Command $command)
         $this->command = $command;
     }
 
-    /**
-     * Executes the command.
-     *
-     * Available execution options:
-     *
-     *  * interactive: Sets the input interactive flag
-     *  * decorated:   Sets the output decorated flag
-     *  * verbosity:   Sets the output verbosity flag
-     *
-     * @param array $input   An array of command arguments and options
-     * @param array $options An array of execution options
-     *
-     * @return int The command exit code
-     */
-    public function execute(array $input, array $options = array())
-    {
-        // set the command name automatically if the application requires
-        // this argument and no command name was passed
-        if (!isset($input['command'])
-            && (null !== $application = $this->command->getApplication())
-            && $application->getDefinition()->hasArgument('command')
-        ) {
-            $input = array_merge(array('command' => $this->command->getName()), $input);
-        }
-
-        $this->input = new ArrayInput($input);
-        if (isset($options['interactive'])) {
-            $this->input->setInteractive($options['interactive']);
-        }
-
-        $this->output = new StreamOutput(fopen('php://memory', 'w', false));
-        if (isset($options['decorated'])) {
-            $this->output->setDecorated($options['decorated']);
-        }
-        if (isset($options['verbosity'])) {
-            $this->output->setVerbosity($options['verbosity']);
-        }
-
-        return $this->statusCode = $this->command->run($this->input, $this->output);
-    }
+     /**
```

Comment:
Fixed. The changes of this PR are separated in https://github.com/symfony/symfony/pull/18710/commits/69dd333eda69554690d586755e9706ffbd000d70